### PR TITLE
Json rpc unit test

### PR DESCRIPTION
**Pull request by taooceros**
Unit Test for JsonRPCPlugins

1. Check whether specific return string is parsed correctly
2. Check whether case is actually insensitive

### CODE REVIEW

File: Flow.Launcher.Core/Flow.Launcher.Core.csproj [hunk 1]
```diff
@@ -58,10 +58,6 @@
     <PackageReference Include="squirrel.windows" Version="1.5.2" />
   </ItemGroup>
   
-  <ItemGroup>
-    <Folder Include="Properties\" />
```
Comment (jjw24) [inline Flow.Launcher.Core/Flow.Launcher.Core.csproj:62]: is this removal intended? we still specify compile include at line 48 of the core csproj file

**Review by jjw24 — CHANGES_REQUESTED** [review]
very nice addition, we should add more tests in general.

for unit tests, could we follow the GivenWhenThen structure please:
https://martinfowler.com/bliki/GivenWhenThen.html

Have a look at the stringmatcher unit tests as an example

File: Flow.Launcher.Core/Flow.Launcher.Core.csproj (hunk 1 above)
Comment (taooceros) [reply Flow.Launcher.Core/Flow.Launcher.Core.csproj:62]: Let me check. I don't know why this is removed.

**Comment by jjw24** [comment]
I removed the unused InternalVisibleTo for Flow.Launcher project. We can add it when it's actually used

**Review by jjw24 — APPROVED** [review]
